README.md

```diff
@@ -4,14 +4,16 @@
 
 This is [zstd](https://github.com/facebook/zstd), packaged for [Zig](https://ziglang.org/).
 
+Compatible with zig `0.14`-`0.16`
+
```

Comment:
A statement like "Compatible with zig 0.16" is misleading because there is no 0.16.0 release (yet). You would have to mention the exact Zig version this has been tested with or say `0.16.0-dev`. This can easily get outdated over time so I'd suggest to remove this statement entirely and let the user test whether it works with their Zig version instead.

Also, projects in allyourcodebase only need to guarantee support for the latest tagged release. Anything beyond that may change over time.